horizontalintegration/xmc-hztl-foundation-2024-q1
# Feat FIS-402 withStandardComponentWrapper Applied to All Components

## What does this PR do and why?

This PR...

1. Applies `withStandardComponentWrapper()` consistently across all components.
2. Updates the Storybook file structure to match the reorganized component file structure.

## Screenshots or screen recordings

N/A

## How to set up and validate locally

1. Check out the branch _feature/FIS-402-withStandardComponentWrapper_.
2. From a command prompt, navigate to _src/hztl-foundation_ and execute `npm run storybook`.
3. Once Storybook loads, confirm that all components are organized correctly and function properly.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
